migrations/versions/166a3402c391_add_extra_fields_to_user.py

```diff
@@ -0,0 +1,47 @@
+"""add extra fields to user
+
+Revision ID: 166a3402c391
+Revises: 6e53eac9ea14
+Create Date: 2024-08-21 14:47:07.012516
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "166a3402c391"
+down_revision = "6e53eac9ea14"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("users", schema=None) as batch_op:
```

Comment:
`schema` is already optional, why set it to `None`?
```suggestion
    with op.batch_alter_table("users") as batch_op:
```

Reply:
Oh, this was part of the autogenerated alembic revision. We actually have some model clean-up to do, so that when we autogenerate a revision it doesn't try to make a bunch of changes. But for now I like your suggestion to remove it.